home/templates/home/question_detail.html

```diff
@@ -0,0 +1,64 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+<!--- Those links are for the icons ---->
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+<body class="main-bg">
+  <div class="question-container">
+  <div class="question-display">
+  <div class="question-title">
+	<img class="rounded-circle question-img" 
+	src="{% get_static_prefix %}home\subjects-images\{{object.subject|lower}}.png">
+	<h3> {{ object.title }} </h3>
+	{% if object.profile.user == user %}
+          <div>
+            <a  style="float:right; margin:3px; color:red;" href="#"><i class="far fa-trash-alt"></i><b>Delete</b></a>
+			 <a  style="float:right; margin:3px; color:black; padding-right:5px;" href="#"><i class="fas fa-edit"></i><b>Edit</b></a>
+          </div>
+    {% endif %}
+  </div>
+  <div class="question-metadata">
+	<small class="text-muted date">{{ object.publish_date|date:"F d, Y" }}</small>
+	<p>by <a class="mr-2" style="color:#6FB5EE" href="#"> <b>{{ object.profile }}</b></a></p>
+  </div>
+    <div class="question-body">
+      <p class="question-content">{{ object.content }}</p>
+	  <p class="question-tags"><b>tags:{% for tag in tags %}
+			#{{ tag.tag }}
+		{% endfor %} </b></p>
+		{% if object.is_edited %}
+		<small class="text-muted"><span>edited</span></small>
+		{% endif %}
+      </div>
+  </div>
+   <div question-footer>
+	  <p class="ans-counter"><b>{{ answersCount }}</b></p>
+	  <div class="btn-group btn-group-toggle  sort-btns" data-toggle="buttons">
+		  <label class="btn active">
+			<input type="radio" name="options" id="option1" autocomplete="off" checked> Newest
+		  </label>
+		  <label class="btn">
+			<input type="radio" name="options" id="option2" autocomplete="off"> Votes
+		  </label>
+	</div>
+	</div>
+</div>
+<div class="answers-container">
+	{% for answer in answers %}
+			<div class="answer-display ">
+				<p class = "left">{{ answer.content }}</p>
+				<div class="answer-meta-data right pl-4">
+					<p>by <a class="mr-2" style="color:#6FB5EE" href="#"> <b>{{ answer.profile }}</b></a></p>
+					<small class="text-muted date">{{ object.publish_date|date:"F d, Y" }}</small>
+					<p style="text-align:center"><br><br>{{ answer.getSumOfLikes }} </p>
```

Comment:
Good job Lior !
I think it would look better to show the number of likes and  the number of dislikes (next to the thumbs up and thumbs down icons) ,instead of showing the subtraction between them